src/components/ConversationIcon.vue

```diff
@@ -90,41 +91,45 @@ export default {
 
 <style lang="scss" scoped>
 .conversation-icon {
-	width: 40px;
-	height: 40px;
-}
-
-.icon:not(.icon-favorite) {
 	width: 44px;
 	height: 44px;
-	line-height: 44px;
-	font-size: 24px;
-	background-color: var(--color-background-darker);
 
-	&.icon-changelog {
-		background-size: 44px;
-	}
-	&.icon-public,
-	&.icon-contacts,
-	&.icon-password,
-	&.icon-file,
-	&.icon-mail {
-		background-size: 20px;
+	&__avatar {
+		// we request 40px avatars, but
```

Comment:
we can also make everything 40 instead. I think this was not an intentional mismatch

Reply:
Well, both are good to me, I really have no preference, but you need to choose :)

Reply:
Let's go for 40 then

Reply:
Well, minimum clickeable area on nextcloud is 44px. SO I'd rather have everything 44.
What do you prefer @jancborchardt ?

Reply:
But visually it looks cleaner with 40px

| 40 | 44 |
|:---------:|:------:|
|![dev skjnldsv com_call_s934fu82](https://user-images.githubusercontent.com/14975046/71891743-8a3e5000-3147-11ea-8a22-031a5175700f.png)|![dev skjnldsv com_call_s934fu82 (1)](https://user-images.githubusercontent.com/14975046/71891749-8c081380-3147-11ea-9969-055e2db61a69.png)|

Reply:
Maybe a 44 by 44 clickable square wrapping a 40px avatar?

Reply:
The avatar component doesn't support it, let's not over complicate things ^^

Reply:
Then I'd also prefer 44px!